raiden/network/rpc/client.py

```diff
@@ -148,6 +148,7 @@ def __init__(
             privkey: bytes,
             gas_price_strategy: Callable = rpc_gas_price_strategy,
             nonce_offset: int = 0,
+            block_num_confirmations: int = 0,
```

Comment:
Have you made sure that in all the integration tests the value of 0 is given? i.e. that for all tests this is 0? Except perhaps one test that could test specifically for confirmations?